### ISSUE TITLE
restream refuses to add youtube because your email is not verified

### ISSUE THREAD

**Issue by mrgodgiven** [WWBN/AVideo#7255]
Greetings Mr @DanielnetoDotCom  When I try to add my youtube channel for restream I get a message that your danielneto@gmail.com email is not verified with google so i can't add my you tube channel for restream. My users are also facing the same pproblem. please help 
![Screenshot (41)](https://user-images.githubusercontent.com/72950503/195601031-829df9a3-10b1-4edb-aa1f-595072b52cda.png)

**Comment by DanielnetoDotCom** [comment]
Yes, I already send the verification process to google a while ago, but they still verifying it. 

(I have checked yesterday)

**Comment by mrgodgiven** [comment]
thanks
